src/js/tracks/text-track-settings.js

```diff
@@ -235,113 +235,112 @@ function setSelectedOption(target, value) {
 
 function captionOptionsMenuTemplate() {
   let template = `<div class="vjs-tracksettings">
-      <div class="vjs-tracksettings-colors">
-        <div class="vjs-fg-color vjs-tracksetting">
-            <label class="vjs-label">Foreground</label>
-            <select>
-              <option value="">---</option>
-              <option value="#FFF">White</option>
-              <option value="#000">Black</option>
-              <option value="#F00">Red</option>
-              <option value="#0F0">Green</option>
-              <option value="#00F">Blue</option>
-              <option value="#FF0">Yellow</option>
-              <option value="#F0F">Magenta</option>
-              <option value="#0FF">Cyan</option>
-            </select>
-            <span class="vjs-text-opacity vjs-opacity">
-              <select>
-                <option value="">---</option>
-                <option value="1">Opaque</option>
-                <option value="0.5">Semi-Opaque</option>
+         <div class="vjs-tracksettings-colors">
+          <fieldset>
+            <legend>Colors</legend>
+            <div class="vjs-fg-color vjs-tracksetting">
+              <label class="vjs-label" for="captions-foreground">Foreground</label>
+              <select id="captions-foreground">
```

Comment:
@gkatsev looking at the `guid()` method, wouldn't a simpler approach be to use the `player.tag.id` (which is unique for each player instance) as part of the IDs in the Caption Player Dialog? It seems like the guid() is going to be unique for each element within the player, which is fine for how it's used within `dom.js`, but perhaps overkill here?

Otherwise, has there been any discussion about having a global `player.uniqueId`?